docs/add-runtime.md

```diff
@@ -0,0 +1,254 @@
+# Creating a new functions runtime 🏃
+
+This document is part of the OpenRuntimes contributors' guide. Before you continue reading this document make sure you have read the [Code of Conduct](https://github.com/open-runtimes/open-runtimes/blob/main/CODE_OF_CONDUCT.md) and the [Contributing Guide](https://github.com/open-runtimes/open-runtimes/blob/main/CONTRIBUTING.md).
```

Comment:
We dont have a code of conduct yet . Maybe you can add that

Reply:
Shall I add the Code of Conduct?

Reply:
Sure @adityaoberai  go ahead